src/plugins/discover/public/application/components/default_discover_table/table_row.tsx

```diff
@@ -142,6 +145,9 @@ const TableRowUI = ({
           <EuiFlexItem>
             <h4
               data-test-subj="docTableRowDetailsTitle"
+              aria-label={i18n.translate('discover.defaultTable.docTableRowDetailsTitleLabel', {
+                defaultMessage: 'Discover table row details',
+              })}
```

Comment:
This being an `h4` and clickable(?) is a violation; let's give it [a role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role) and change the message to "Expand document".

Reply:
actually i think this is just a non-interactive <h4>, it is not a button. Will just delete the aria-label. @AMoo-Miki 

<img width="520" alt="Screenshot 2024-06-07 at 3 35 42 PM" src="https://github.com/opensearch-project/OpenSearch-Dashboards/assets/43937633/c6629a1d-a886-4fa1-8589-0e72ba5e5e40">